sql/encrypted-connections.md

```diff
@@ -5,6 +5,56 @@ category: user guide
 
 # 使用加密连接
 
+## 摘要
```

Comment:
Looks like this content is already covered by later content?

Reply:
For what I mean, this part is much like TL;DR

Reply:
But looks it's still too long 🤔🤣 I have never seen a TL;DR in so many lines